airflow/models/dag.py

```diff
@@ -3874,10 +3902,14 @@ def _run_task(ti: TaskInstance, session):
     try:
         ti._run_raw_task(session=session)
         session.flush()
-        log.info("%s ran successfully!", ti.task_id)
+        if ti.state == TaskInstanceState.DEFERRED:
+            log.info("%s has deferred", ti.task_id)
+        else:
+            log.info("%s ran successfully!", ti.task_id)
     except AirflowSkipException:
         log.info("Task Skipped, continuing")
     log.info("*****************************************************")
+    return ti
```

Comment:
Why does this need to be changed? Isn’t this `ti` the same as the `ti` passed in as argument?

Reply:
yeah not require